common/djangoapps/course_modes/helpers.py

```diff
@@ -79,15 +72,9 @@ def _enrollment_mode_display(enrollment_mode, verification_status, course_id):
     course_mode_slugs = [mode.slug for mode in CourseMode.modes_for_course(course_id)]
 
     if enrollment_mode == CourseMode.VERIFIED:
-        if (
-            settings.FEATURES.get('ENABLE_INTEGRITY_SIGNATURE')
-            or verification_status in [VERIFY_STATUS_NEED_TO_VERIFY, VERIFY_STATUS_SUBMITTED, VERIFY_STATUS_APPROVED]
-        ):
-            display_mode = DISPLAY_VERIFIED
-        elif DISPLAY_HONOR in course_mode_slugs:
+        display_mode = DISPLAY_VERIFIED
+        if DISPLAY_HONOR in course_mode_slugs:
             display_mode = DISPLAY_HONOR
-        else:
-            display_mode = DISPLAY_AUDIT
```

Comment:
There's no way to have this display mode in the same code path when we do not require IDV for a course anymore, so I've chosen to remove it.